extensions/unicode-symbols/scripts/generate-dataset.ts

```diff
@@ -59,20 +63,45 @@ function getCharactersByCodeRange(startCode: number, endCode: number) {
   return characters;
 }
 
+const mapCodeToName = (char: Character): Character => {
+  const map: Record<number, string> = { 63743: "APPLE LOGO" };
```

Comment:
Can we add a comment here explaining what is supposed to go into `map`? (Or extract it from here and move it up a la' constant).